### PR TITLE
Doc: Mention how to add libary to project in project files doc.

### PR DESCRIPTION
**Pull request by Kakadu**
I don't know though how to check that change is OK for this markup. It's not obvious how to do that when looking at `doc/` contents.

### CODE REVIEW

File: doc/dune-files.rst [hunk 1]
```diff
@@ -247,6 +247,9 @@ It contains the following fields:
   <url>)``, ``(documentation <url>)`` are the same (and take precedence over)
   the corresponding global fields. These fields are available since Dune 2.0.
 
+Adding libraries to different projects is done via  ``public_name`` field. See 
+:ref:`library` section for details.
```
Comment (diml) [inline doc/dune-files.rst]: Do you mean adding libraries to packages?

**Comment by diml** [comment]
You can do `make doc` or `make livedoc`. You'll need to install the `sphinx` pyhton package as well as the readthedocs theme (I can't remember the name of the package for this one).

File: doc/dune-files.rst (hunk 1 above)
Comment (Kakadu) [reply doc/dune-files.rst]: Yeah, you are right.

**Comment by Kakadu** [comment]
> You can do `make doc` or `make livedoc`. You'll need to install the `sphinx` pyhton package as well as the readthedocs theme (I can't remember the name of the package for this one).

I checked docs generation. It looks as I expect

**Comment by diml** [comment]
Thanks!

**Pull request merged**